extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -122,10 +140,10 @@ public function addPowerUserSetOption( $sProperty ) {
 	public function addPowerUserAddGroup( $sProperty ) {
 		if ( in_array( $sProperty, self::$aPowerUserProperties )
 			&& $this->bUseGroups
-			&& !in_array( self::GROUP_NAME, \UserRights::getGlobalGroups( $this->oUser ) )
+			&& !in_array( self::GROUP_NAME, $this->userPermissions()->getExplicitGlobalUserGroups( $this->oUser->getId() ) )
```

Comment:
it would be much more readable if we could write something like:
`$this->userPermissions()->hasGlobalGroup( self::GROUP_NAME )`
can we add a handful of helper methods to the user permissions service class and use them wherever possible?